src/components/Camera/StandardCamera.js

```diff
@@ -167,105 +174,195 @@ const StandardCamera = ( ): Node => {
+        { hasFlash && (
+          renderFlashButton()
+        )}
+        <Pressable
+          className={`${cameraOptionsClassName}`}
+          onPress={flipCamera}
+          accessibilityLabel={t( "Camera-button-label-switch-camera" )}
+          accessibilityRole="button"
+          accessibilityState={{ disabled: false }}
+        >
+          <INatIcon
+            name="rotate"
+            color={colors.white}
+            size={20}
+            testID="camera-button-label-switch-camera"
+          />
+        </Pressable>
+        <Pressable
+        // eslint-disable-next-line max-len
+          className="bg-white rounded-full h-[60px] w-[60px] justify-center items-center"
+          onPress={takePhoto}
+          accessibilityLabel={t( "Navigate-to-observation-edit-screen" )}
+          accessibilityRole="button"
+          accessibilityState={{ disabled: disallowAddingPhotos }}
+          disabled={disallowAddingPhotos}
+        >
+          <View className="border-[1.64px] rounded-full h-[49.2px] w-[49.2px]" />
+        </Pressable>
+        {photosTaken && (
           <Pressable
-            onPress={flipCamera}
-            accessibilityLabel={t( "Camera-button-label-switch-camera" )}
+            className={checkmarkClass}
+            onPress={navToObsEdit}
+            accessibilityLabel={t( "Navigate-to-observation-edit-screen" )}
             accessibilityRole="button"
+            accessibilityState={{ disabled: false }}
+            disabled={false}
           >
-            <Avatar.Icon
+            <INatIcon
+              name="checkmark"
+              color={colors.white}
+              size={20}
               testID="camera-button-label-switch-camera"
-              size={40}
-              icon="flip"
-              style={{ backgroundColor: colors.darkGray }}
             />
           </Pressable>
+        )}
+        <View className={`${cameraOptionsClassName}`}>
+          <CloseButton
+            handleClose={cameraPreviewUris.length > 0 && handleBackButtonPress}
+            size={18}
+          />
         </View>
-        <View className="h-[89px] flex-row justify-between items-center mx-7">
-          <CloseButton handleClose={cameraPreviewUris.length > 0 && handleBackButtonPress} />
-          <IconButton
-            icon="camera"
+      </View>
+    );
+  };
+
+  return (
+    <View className="flex-1 bg-black">
+      <StatusBar barStyle="light-content" />
+      <PhotoPreview
+        photoUris={cameraPreviewUris}
+        setPhotoUris={setCameraPreviewUris}
+        savingPhoto={savingPhoto}
+        screenBreakpoint={breakpoint}
+      />
+      <View className="relative flex-1">
+        {device && <CameraView device={device} camera={camera} orientation={imageOrientation} />}
+        <FadeInOutView savingPhoto={savingPhoto} />
+        {breakpoint === "sm" || breakpoint === "md" ? renderSmallScreenCameraOptions()
```

Comment:
The `breakpoint === "sm" || breakpoint === "md"` is repeated and its meaning is not immediately apparent, so you might want to pull that out into a variable so you're not repeating yourself and it's more readable, e.g. `const isSmallScreen = ["sm", "md"].includes( breakpoint );`.

Also, my simulated iPhone and my Pixel 3 both seem to have the `lg` breakpoint in portrait orientation, so they always get the large screen layout. Shouldn't we consider `lg` to be a small screen layout?